app/services/food_data_service.rb

```diff
@@ -0,0 +1,11 @@
+class FoodDataService
+
+  def self.search_foods(query)
+    conn = Faraday.new(url: "https://api.nal.usda.gov") do |f|
+      f.params["api_key"] = ENV["FOOD_KEY"]
+    end
+
+    response = conn.get("/fdc/v1/foods/search?query=#{query}")
```

Comment:
I would set my query param in a block here the same way that you did in the connection object above. That's more readable and maintainable.